code-challenges.test.js

```diff
@@ -15,35 +15,127 @@
+// Use .sort to produce an a new arrangement using the random key. 
+// Return expected output. 
+
+const firstShuffle = (shuffleArray) => {
+    return shuffleArray.shift() + shuffleArray.map(value => ({ value, sort: Math.random() })) + shuffleArray.sort((a, b) => a.sort - b.sort)
+    
+}
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// (N/A - Nothing showed up when I ran the test, just said PASS but only as if I had the describe method aka Part A info only...it didn't register my logic.)
+
+// ** NOTES TO INSTRUCTOR **: I wasn't sure if I used the correct describe method setup based on the linked resource but I made my best guest and received a PASS for it. However, when it came to the second part of the question (the logic) as you can read above in the "RAN TEST" section it didn't register in Jest, it doesn't run the test for my logic for some reason. Any clarity on that? Regarding the logic, I wasn't sure if I should use Math.floor(Math.random() as I thought they may be mostly used for integers but I wasn't sure how else to randomize the elements of the array without it and my research wasn't providing many alternatives either. Side question should I have used .slice? 
+
 
 // --------------------2) Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("minToMax", () => {
+    it("takes an array of numbers and returns an array of the minimum and maximum numbers in that order", () => {
+        const nums1 = [3, 56, 90, -8, 0, 23, 6]
+        // Expected output: [-8, 90]
+        const nums2 = [109, 5, 9, 67, 8, 24]
+        // Expected output: [5, 109]
+        expect(minToMax([3, 56, 90, -8, 0, 23, 6])).toEqual([-8, 90])
+        expect(minToMax([109, 5, 9, 67, 8, 24])).toEqual([5, 109])
+
+    })
+})
 
-const nums1 = [3, 56, 90, -8, 0, 23, 6]
-// Expected output: [-8, 90]
-const nums2 = [109, 5, 9, 67, 8, 24]
-// Expected output: [5, 109]
+//* RAN TEST (RESULTS BELOW): *//
 
+// ReferenceError: minToMax is not defined
 
 // b) Create the function that makes the test pass.
 
+//Goal: Takes an array of numbers and returns an array of the minimum and maximum numbers in that order
+
+//Pseudocode:
+// Declare a function called minToMax 
+// Set the parameter as numericalArray.
+// Use .sort method to sort array from ascending order by specifying comparison function correctly. 
+// Return desired output. 
+
+// const minToMax= (numericalArray) => {
+//     numericalArray.sorts(function(a,b){return a-b;})
+//     return a - b; 
+// }
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// FAIL 
+// TypeError: numericalArray.sorts is not a function. 
+
+// ** NOTES TO INSTRUCTOR **: I was a bit surprised to see this error as a result of running my test, I thought my logic was...well logical. What did I do incorrectly? (Hence why my code is commented out here. )
```

Comment:
I understand your logic - the structure is a little off.  I think you misread the question.  Your logic will put them in order, but it wants only the lowest and the highest number returned.